src/ContainerDetails.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
This file must be named .jsx, to apply correct babel and eslint rules. It's not "classic" non-ES6 JavaScript. Same applies to the other *.js files in this PR.